src/application/services/useNote.ts

```diff
@@ -92,6 +97,20 @@ export default function (options: UseNoteComposableOptions): UseNoteComposableSt
    */
   const router = useRouter();
 
+  /**
+   * Title identifier
+   */
+  const limitChars = 50;
```

Comment:
variable naming is unclear for its scope (`useNote`)

Reply:
Rename variables